docs/usage.rst

```diff
@@ -1,34 +1,38 @@
 Using ``pyramid_nacl_session``
 ==============================
 
-Generating a Random Secret
---------------------------
+Installation
+------------
 
-First, you need to arrange to have a 32-byte random secret available:
+Install using setuptools, e.g. (within a virtualenv)::
 
-.. code-block:: python
+  $ pip install pyramid_nacl_session
 
-   from pyramid_nacl_session import generate_secret
+Setup
+-----
 
-   SECRET = generate_secret(as_hex=False)
+Once :mod:`pyramid_nacl_session` is installed, you typically use the 
+``config.include`` mechanism to include it into your Pyramid project's 
+configuration. In your Pyramid project's ``__init__.py``:
 
-Configure a Session Factory
----------------------------
+.. code-block:: python
 
-Use the generated secret to configure a session factory:
+   config = Configurator(.....)
+   config.include('pyramid_nacl_session')
 
-.. code-block:: python
+Alternately, instead of using the Configurator's ``include`` method, you can
+activate :mod:`pyramid_nacl_session` by changing your application's ``.ini`` 
+file, use the following line:
 
-   from pyramid.session import BaseCookieSessionFactory
-   from pyramid_nacl_session import EncryptedCookieSessionFactory
+.. code-block:: ini
 
-   def includeme(config):
-       factory = EncryptedCookieSessionFactory(SECRET)  # other config ad lib.
-       config.set_session_factory(factory)
+   pyramid.includes = pyramid_nacl_session
 
-Sharing the Secret Across Instances
------------------------------------
+Usage
+-----
 
+Generate a secret key
+`````````````````````
```

Comment:
Please use heading underlines consistent with the levels in our [Style Guide](https://docs.pylonsproject.org/projects/docs-style-guide/#headings) and adopted from [Python](https://devguide.python.org/documenting/#sections). In this case, `^`.